ui/src/app/views/admin.html

```diff
@@ -18,12 +18,12 @@
 
     <table id="table" class="table table-hover table-bordered signup-requests-table">
         <tbody>
-        <tr ng-repeat="data in signup_requests">
+        <tr ng-repeat="data in signup_requests" ng-hide="number_to_approve == $index">
             <td data-title="ID">{{$index + 1}}</td>
             <td data-title="Name">{{data.user_name}}</td>
             <td data-title="Email">{{data.email}}</td>
             <td id="approve-btn">
-              <md-button class="md-raised" ng-click="approve(data.user_name)">
+              <md-button class="md-raised" ng-click="approve(data.user_name); number_to_approve = $index">
```

Comment:
Having logic in both view and controller will make the code hard to debug and maintain. Let's move this inside the controller

Reply:
I've shifted the logic to the controller now